snap/quota/resources.go

```diff
@@ -176,8 +176,13 @@ func (qr *Resources) validateJournalQuota() error {
 	}
 
 	if qr.Journal.Rate != nil {
-		if qr.Journal.Rate.Count <= 0 || qr.Journal.Rate.Period < time.Microsecond {
-			return fmt.Errorf("journal quota must have a rate count larger than zero and period at least 1 microsecond (minimum resolution)")
+		// The only invalid numbers for the count are those less than 0
```

Comment:
This comment doesn't provide more information than the line it's documenting

Reply:
Fixed!